types/canonical.go

```diff
@@ -51,26 +51,17 @@ func CanonicalizeProposal(chainID string, proposal *tmproto.Proposal) tmproto.Ca
 	}
 }
 
-func GetVoteExtensionToSign(ext *tmproto.VoteExtension) *tmproto.VoteExtensionToSign {
-	if ext == nil {
-		return nil
-	}
-	return &tmproto.VoteExtensionToSign{
-		AppDataToSign: ext.AppDataToSign,
-	}
-}
-
 // CanonicalizeVote transforms the given Vote to a CanonicalVote, which does
-// not contain ValidatorIndex and ValidatorAddress fields.
+// not contain ValidatorIndex and ValidatorAddress fields, or any fields
+// relating to vote extensions.
 func CanonicalizeVote(chainID string, vote *tmproto.Vote) tmproto.CanonicalVote {
```

Comment:
Do we need a similar function for canonical vote extensions?

Reply:
We now have `CanonicalizeVoteExtension`